### ISSUE TITLE
Supersampling technique

### ISSUE THREAD

**Issue by emredurukn** [jonmagic/grim#37]
Hi guys, I need to improve the quality of the images I get from the PDF. When I was researching from the internet, I encountered the supersampling technique. People try this technique in the following way.

`convert -density 288 image.pdf -resize 25% resultimage.png`

-resize option works by default before conversion. How can I run this line with grim gem?

**Comment by jonmagic** [comment]
I think you would do something like this:

```ruby
pdf = Grim.reap("/path/to/pdf")
pdf[0].save('/path/to/image.png', {
  :width => '25%',
  :density => 288,
})
```

**Comment by emredurukn** [comment]
I've tried a lot of ways to apply this technique. I convert the pdf with high resolution and then called it with lower resolution in the part where we used the project.

the low resolution i talked about is the resolution that is scaled according to the screen we use in the project.

As a result, I applied the steps of SuperSampling on two different parts of the project.

The first part was choosing a high resolution while converting the pdf in the grim gem. The second part was already in the project.
